### ISSUE TITLE
Cannot find module html-entities.json

### ISSUE THREAD

**Issue by chriswolmarans** [Faire/mjml-react#57]
Hi All, thanks so much for taking on this fresh direction on this fork. I tried out the alpha mentioned in [issue 36](https://github.com/Faire/mjml-react/issues/36) as I ran into the same problem and got the following error:

```
Cannot find module './html-entities.json' from '../../node_modules/@faire/mjml-react/dist/src/utils/html-entities.js'
Require stack:
/app/node_modules/@faire/mjml-react/dist/src/utils/html-entities.js
/app/node_modules/@faire/mjml-react/dist/src/utils/index.js
/app/node_modules/@faire/mjml-react/utils.js
```
I’m not sure if I’m using the render function correctly, `import {render} from ‘@faire/mjml-react/utils’;` or if I’m supposed to implement my own. 

Appreciate any help on this. Thank you!

**Comment by IanEdington** [comment]
Thanks for trying out the alpha and reporting this @chriswolmarans

It looks like we're not packaging that file.

@hchiufaire would you be able to look into bundling the `html-entities.json` file with both the `esm` and `cjs` packages?

**Comment by hchiufaire** [comment]
Hi @chriswolmarans,

This issue has now been addressed in v3.0.0, which has just been released to npm.

**Comment by chriswolmarans** [comment]
@hchiufaire thanks for the heads up, appreciate the help, will try it again.
